Earthfile

```diff
@@ -1,8 +1,8 @@
 VERSION 0.7
 
-ARG ALPINE_VERSION=3.19
-ARG GO_VERSION=1.22
-ARG LINTER_VERSION=v1.57.2
+ARG --global ALPINE_VERSION=3.19
+ARG --global GO_VERSION=1.22
+ARG --global LINTER_VERSION=v1.57.2
```

Comment:
I'm not sure this is necessary, is it?

Reply:
might not be necessary, i vaguely recall running into issues with eathly 0.8+ without it i think